app/services/katello/pulp3/repository/ansible_collection.rb

```diff
@@ -8,7 +8,7 @@ def remote_options
           if root.url.blank?
             super
           else
-            common_remote_options.merge(url: root.url,
+            common_remote_options.merge(url: root.url.chomp('/').concat('/'),
                                         requirements_file: root.ansible_collection_requirements.blank? ? nil : root.ansible_collection_requirements,
```

Comment:
Not sure about this or if we should be adding the trailing slash if things slip through cracks!